sdk/core/azure-core/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Features Added
 
+- A keyword argument `enable_cae` was added to the `get_token` method of the `TokenCredential` protocol.  #31012
+- `BearerTokenCredentialPolicy` and `AsyncBearerTokenCredentialPolicy` now accept `enable_cae` keyword arguments in their constructors. This is used in determining if Continuous Access Evaluation (CAE) should be enabled for each `get_token` request.  #31012
```

Comment:
Probably good to link out to the CAE explainer [here](https://learn.microsoft.com/en-us/azure/active-directory/conditional-access/concept-continuous-access-evaluation). Would adding some information in to the README  also help?

Reply:
Good idea, I updated the `CLIENT_LIBRARY_DEVELOPER.md` file with more info as well.